nilearn/_utils/niimage.py

```diff
@@ -0,0 +1,127 @@
+import copy
+import gc
+import collections
+
+import nibabel
+
+
+def _get_shape(img):
```

Comment:
General remark: there appears to be quite some inconsistency in when and how this function is used in nilearn code.